package.json

```diff
@@ -42,14 +45,27 @@
     }
   },
   "dependencies": {
+    "@blueprintjs/core": "^4.6.0",
+    "@blueprintjs/icons": "^4.3.1",
+    "@blueprintjs/select": "^4.5.3",
     "@datawheel/canon-core": "^0.25.12",
     "async-throttle": "^1.1.0",
+    "babel-preset-react": "^6.24.1",
+    "compose-function": "^3.0.3",
     "countryjs": "^1.8.0",
-    "d3plus-react": "^0.6.1",
+    "crypto-js": "^4.1.1",
+    "d3plus-export": "^1.1.0",
+    "d3plus-react": "^1.1.1",
+    "i18next-http-backend": "^1.4.1",
+    "next-translate": "^1.5.0",
```

Comment:
Will the trivia support multiple languages? If so, canon comes with translations so we wouldn't need an external package for that.